apps/web/src/server/app/validation/validation.controller.js

```diff
@@ -94,5 +108,85 @@ export async function postAppealOutcome(request, response, next) {
 		});
 	}
 
+	if (reviewOutcome === '3') {
+		return response.redirect(`/validation/outcome-incomplete/${appealId}`);
+	}
+
 	return response.redirect(`/validation`);
 }
+
+/**
+ * GET the outcome incomplete page.
+ *
+ * @param {object} request - Express request object
+ * @param {object} response - Express request object
+ * @param {Function} next  - Express function that calls then next middleware in the stack
+ * @returns {void}
+ */
+export async function getOutcomeIncomplete(request, response, next) {
+	const appealId = request.param('appealId');
+
+	const [error, appealData] = await to(findAppealById(appealId));
+
+	const {
+		body: { errors = {}, errorSummary = [] }
+	} = request;
+
+	if (Object.keys(errors).length > 0) {
+		return response.render('validation/appeal-details', {
+			errors,
+			errorSummary,
+			appealData
+		});
+	}
+
+	return response.render('validation/outcome-incomplete', {
+		appealId
+	});
+}
+
+/**
+ * POST the outcome incomplete page
+ * It will render the incomplete outcome check and confirm page
+ *
+ * @param {object} request - Express request object
+ * @param {object} response - Express request object
+ * @param {Function} next  - Express function that calls then next middleware in the stack
+ * @returns {void}
+ */
+export function postOutcomeIncomplete(request, response, next) {
```

Comment:
Change the name of the function to match the GET one once you rename the GET one.